.github/workflows/test.yml

```diff
@@ -250,6 +250,11 @@ jobs:
         name: Test hls-explicit-fixity-plugin test suite
         run: cabal test hls-explicit-fixity-plugin --test-options="$TEST_OPTS" || LSP_TEST_LOG_COLOR=0 LSP_TEST_LOG_MESSAGES=true LSP_TEST_LOG_STDERR=true cabal test hls-explicit-fixity-plugin --test-options="$TEST_OPTS"
 
+      ## version needs to be limited since the tests depend on cabal-fmt which only builds using specific ghc versions
```

Comment:
Sadly, I think this means we won't test that the plugin itself builds on other GHC versions, right? Which we do want to assert.

Reply:
I think we do build the plugin, but the plugin itself has no GHC specific changes, so it doesn't matter too much in my opinion. Alternatively, we can modify the build environment to always provide a cabal-fmt, but I much rather would create a ticket for it and rework CI before doing something like that.

Reply:
I'm confused: when do we build it? The condition here means we only do this step on 8.10.7, and I'm pretty sure this is the only step that builds things.

Reply:
We only build and run the test-suite for GHC 8.10.7, but the plugin itself *should* be built here: https://github.com/haskell/haskell-language-server/blob/master/.github/workflows/test.yml#L109

Reply:
Ah yes, you are right.